### ISSUE TITLE
NVDA stops working in all browsers besides Edge - Windows 10

### ISSUE THREAD

**Issue by landmine** [nvaccess/nvda#5962]
When tabbing through elements in a webpage, NVDA announces 'unknown' instead of correctly identify elements.  This machine is a Windows 10 laptop, and I have been using NVDA with Firefox almost daily since January without problems.  It came with Windows 10 Home from the factory, so this was not an upgrade.  The problem started on last Thursday 5/12.  I believe that it also coincided with a Windows update.  But I have uninstalled all updates from that date and a few days before, and the problem persists.

NVDA works correctly on the desktop, Windows start menu, and most applications.  MSN Money reads perfectly, and NVDA is working with Edge.  But it fails to work with IE11, Thunderbird, Chrome and Firefox ever since this problem has started.  As I move through these applications, NVDA reads 'unknown' for each element (Chrome, FF, Thunderbird) or in IE11 NVDA is silent.  This is the case when NVDA is installed on the machine or I use the portable version (both 2016.1)

This problem is not present using JAWS.  I've tried booting into safe mode to test, but I get a message indicating that no wave device is available to play sound.  Please let me know what you think and if there is any other info I can provide.

**Comment by jcsteh** [comment]
Duplicate of #2807.

**Comment by landmine** [comment]
NVDA is working again in all browsers after running this in a cmd window as admin:

`for %x in (c:\windows\system32\*.dll) do regsvr32 /s %x`

effectively re-registering all components in the system32 folder.

**Comment by derekriemer** [comment]
@MichaelDCurran @jcsteh should we document this somewhere?


On 5/19/2016 5:20 AM, Christopher W. Land wrote:
>
> NVDA is working again in all browsers after running this in a cmd 
> window as admin:
>
> |for %x in (c:\windows\system32\*.dll) do regsvr32 /s %x|
>
> effectively re-registering all components in the system32 folder.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/nvaccess/nvda/issues/5962#issuecomment-220296454>
>

-- 
------------------------------------------------------------------------


    Derek Riemer

  * Department of computer science, third year undergraduate student.
  * Proud user of the NVDA screen reader.
  * Open source enthusiast.
  * Member of Bridge Cu
  * Avid skiier.

Websites:
Honors portfolio <http://derekriemer.com>
Awesome little hand built weather app! 
<http://django.derekriemer.com/weather/>

email me at derek.riemer@colorado.edu <mailto:derek.riemer@colorado.edu>
Phone: (303) 906-2194

**Comment by Brian1Gaff** [comment]
the last time I had the browsers stopping working it was due to uninstalling 
a demo version of Microsoft office and then when windows update day came it 
downloaded updates to the version I'd uninstalled.
 There was at that time a fixit utility for it on the Microsoft site. That 
cured it once and for all, otherwise every time a windows update contained 
an office update it got screwed up all over again.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----
